tasks/main.yml

```diff
@@ -1,2 +1,17 @@
 ---
-# tasks file for .
\ No newline at end of file
+- name: Update Cache
+  apt:
+    update_chache: yes
+  changed_when: false
+
+- name: Install redis-server
+  apt:
+    name: redis-server
+
+- name: Copy Redis conf 
```

Comment:
Is there a service that should be restarted if this file changes?

Reply:
you are right forgot about that

Reply:
done